doc/source/getting_started/wsl.rst

```diff
@@ -15,37 +15,42 @@ Windows Server 2019. For more information, see:
 This page walk you through the installation of WSL on Windows and then
 show how to use it together with MAPDL, PyMAPDL, and `Docker <https://www.docker.com/>`_.
 
-.. caution::
-   This approach hasn't been fully tested with a VPN connection. If you
-   experience any problems connecting WSL to the internet, try to
-   disconnect from the VPN.
 
+.. note::
+   Updating this guide is doing a best effort basis. Due to WSL being under constant
+   development this guide might be outdated without any notice. 
+   If you find any issue or problem, feel free to
+   `open an issue in the GitHub repository <pymapdl_issues>`_.
```

Comment:
```suggestion
   Because WSL is under constant development, keeping this guide updated is difficult. If you find
   any issues or have questions related to WSL, feel free to `open an issue in the GitHub repository <pymapdl_issues>`_.
```